src/common/filesystembase.cpp

```diff
@@ -478,4 +478,20 @@ bool FileSystem::isLnkFile(const QString &filename)
     return filename.endsWith(".lnk");
 }
 
+bool FileSystem::isJunction(const QString &filename)
+{
+#ifdef Q_OS_WIN
+    WIN32_FIND_DATA findData;
+    HANDLE hFind = FindFirstFileEx((const wchar_t *)filename.utf16(), FindExInfoBasic, &findData, FindExSearchNameMatch, NULL, 0);
+    if (hFind != INVALID_HANDLE_VALUE)
+        FindClose(hFind);
```

Comment:
and return false?  
I'm just afreak the findData is left uninitialized. (UB)

Reply:
Yes, seems safer that way.